File: 백준/Bronze/2609. 최대공약수와 최소공배수/최대공약수와 최소공배수.cc
#include <iostream>

using namespace std;

int main(){
    int num1,num2;
    cin >> num1 >> num2;
    int max = 0;
    for(int i=1;i<=num2;i++){
        if(num2%i==0){
            if(num1%i==0){
                if(max<i){
                    max = i;
                }
            }
        }
    }
    cout << max << endl;
    cout << num1*num2/max << endl;
}

File: 프로그래머스/1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.js
function solution(n, m) {
    let num1 = n, num2 = m;
    if(n < m){
        var tmp = m;
        m = n;
        n = tmp;
    }
    while(num2>0){
        let r = num1 % num2;
        num1 = num2;
        num2 = r;
    }
    var answer = [num1, n*m/num1];
    return answer;
}